templates/Web/Projects/VueDefault/README_postaction.md

```diff
@@ -0,0 +1,56 @@
+﻿## Getting Started
+
+In the root directory of the project...
+
+1. Install node modules `yarn install` or `npm install`.
+2. Start development server `yarn start` or `npm start`.
+
+## Next Steps
+//{[{
+### Adding a New Page
+
+1. Create a file in `/src/views` with your Vue Template.
+2. Add a route for your page to `/src/router/index.js`.
+3. Add a button to the navigation bar in `/src/components/NavBar.vue`.
+//}]}
+### Deployment
+
+The generated templates can be deployed to Azure App Service using the following steps:
+
+1. In the root directory of the project `yarn build` or `npm build` to create a build folder.
+2. Move the build folder inside the server folder.
```

Comment:
Seems like this step is automated, should we remove this?

Reply:
This content is more so for if they want to additionally add to the project post gen. So lets say i want to create a new page and have it show up. How would I go about that.

Reply:
I was more referring to this step `2. Move the build folder inside the server folder.` since we have `mv build server` in the package.json which moves the build folder (correct me if im misunderstanding something)

Reply:
Yea sure, separate PR to change all of them would be good. 

I actually don't think angular has the `mv build server` right now